runtime/rococo/src/lib.rs

```diff
@@ -793,6 +799,40 @@ impl mmr_common::Config for Runtime {
 	type ParachainHeads = Paras;
 }
 
+parameter_types! {
+	// This is a pretty unscientific cap.
+	//
+	// Note that once this is hit the pallet will essentially throttle incoming requests down to one
+	// call per block.
+	pub const MaxRequests: u32 = 4 * HOURS as u32;
+
+	// Number of headers to keep.
+	//
+	// Assuming the worst case of every header being finalized, we will keep headers at least for a
+	// week.
+	pub const HeadersToKeep: u32 = 7 * DAYS as u32;
+}
+
+pub type RococoGrandpaInstance = ();
+impl pallet_bridge_grandpa::Config for Runtime {
+	type BridgedChain = bp_rococo::Rococo;
+	type MaxRequests = MaxRequests;
+	type HeadersToKeep = HeadersToKeep;
+
+	// TODO [#391]: Use weights generated for the Rococo runtime instead of Rialto ones.
```

Comment:
looking at the weightinfo in this file, we don't seem to care
does 391 refer to https://github.com/paritytech/parity-bridges-common/issues/391?
I would suggest either remove the comment or create an issue in polkadot

Reply:
I'd keep the weights to be able to better monitor relayer costs. Indeed if no other component cares about weights on Rococo and there are no plans to introduce benchmarking, we can safely remove the comment.